source/config/configSpec.py

```diff
@@ -362,6 +362,9 @@
 		__many__ = string(default="")
 	[[ui]]
 		confirmDisconnectAsFollower = boolean(default=True)
+[captionLocal]
+	loadModelWhenInit = boolean(default=false)
+	localModelPath = string(default=models/Xenova/vit-gpt2-image-captioning)
```

Comment:
I think we should go with the name `defaultModelPath`.
In future we may want a new config item with a list of downloaded models to chose from, and so this config item will be used as the current default.

```suggestion
	defaultModelPath = string(default="models/Xenova/vit-gpt2-image-captioning")
```